pkg/assert/validate.go

```diff
@@ -50,20 +55,21 @@ func runAssertion(
 	errorsChan chan<- error,
 ) error {
 	for {
+		res, err := exec(cmd)
+		if err != nil {
+			errorsChan <- err
+			return fmt.Errorf("error from runCmd: %s\n %s", res, err)
```

Comment:
Can we clarify and log which cmd causes this error?

Reply:
sure